staging/src/k8s.io/kubectl/pkg/util/templates/help_flags_printer.go

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Please include the change for the `--output` character separator to make those lines play nice.